src/pages/sigcse2024/index.tsx

```diff
@@ -118,6 +187,17 @@ export default function SIGCSE2024() {
               their own laptops during the event.
             </p>
 
+            <div className="container-fluid py-2">
+              <div className="d-flex flex-row justify-content-center flex-wrap">
+                <Link
+                  href="https://us.prairielearn.com/pl/public/course/1305/questions"
```

Comment:
I'll assume we're ok changing this link before all the questions are in place?

Unrelated to this PR, but: I have the same background fear about now needing to support these questions forever, in case people start using them in their course. I know there's not much we can do about that at this point though.

Reply:
yes, it is okay to have just some of the questions there. Firas will be taking care of updating the repo, and I am not sure when he will be done with that. I need to have all my "to do" items done before tomorrow since I will not have any time to spare during the conference.